lib/src/component/abstract_transition.dart

```diff
@@ -187,16 +190,19 @@ abstract class AbstractTransitionComponent<T extends AbstractTransitionProps, S
       skipCount = 0;
     }
 
```

Comment:
Potentially, but wouldn't that be accomplishing the same as what I linked above, albeit in a subset of cases?